### PR TITLE
add edwardchalstrey1 to contributors

### PR DESCRIPTION
**Pull request by edwardchalstrey1**
Testing I can commit, PR, merge, etc

### CODE REVIEW

**Review by tomalrussell — APPROVED** [review]
:100:

File: .all-contributorsrc [hunk 1]
```diff
@@ -79,7 +79,17 @@
...
+    {
+      "login": "edwardchalstrey1",
+      "name": "edwardchalstrey1",
+      "avatar_url": "https://www.gravatar.com/avatar/86760965588048d266a6bf9f50f18a16",
+      "profile": "https://github.com/edwardchalstrey1",
+      "contributions": [
+        "code",
+        "doc"
+      ]
+    },
```
Comment (matkoniecz) [inline .all-contributorsrc:92]: trailing comma is added

Unfortunately the JSON specification does not allow a trailing comma. 

Can we ignore this bad design here?

**Comment by edwardchalstrey1** [comment]
Thanks @matkoniecz ! Although I think there's something more I need to add to get myself to show up on the readme, will investigate 🧐

**Comment by tomalrussell** [comment]
Ah - I think technically we should have asked 

@all-contributors please add edwardchalstrey1 for code and docs

**Comment by tomalrussell** [comment]
@all-contributors please add @edwardchalstrey1 for code and docs

**Comment by tomalrussell** [comment]
hmm - ed you might need to edit in your own line in the README, or revert this and go again..

While we're here:

@all-contributors please add @matkoniecz  for code and docs

**Comment by edwardchalstrey1** [comment]
@tomalrussell I see! I shouldn't have tried to do work a bot could do for me! 😆

**Comment by edwardchalstrey1** [comment]
Ok, I reverted this PR, now to try:

@all-contributors please add edwardchalstrey1 for code and docs

**Comment by edwardchalstrey1** [comment]
@all-contributors please add @mdsimpson42 for code and docs

**Pull request merged**